docs/tutorial/index.md

```diff
@@ -2,16 +2,16 @@
 title: Gatsby.js Tutorial
 ---
 
-We’re so happy you decided to try using Gatsby. This tutorial has an initial introduction plus seven more parts that'll walk you from getting started developing and building Gatsby sites to deploying a finished and polished high performance static PWA.
+Welcome to Gatsby! We’re glad you’re here. The goal of this tutorial is to guide you through setting up and deploying your first Gatsby site using a starter template. As we walk through that process, we’ll introduce some more general web development topics, and go over the underlying structure of a Gatsby site. 
 
-Note: If you prefer a quick start rather than step-by-step tutorial, see the [getting started page](/docs/).
+The full tutorial is intended to be as accessible as possible to people without much web development experience (yet!) — no need to be an expert. If you prefer to jump straight to code, feel free to skip the step-by-step tutorial and see the [quick start](/docs/) page.
 
-This tutorial is for _everyone_! You do not need to be a programmer or React.js expert. We'll walk you through things.
+## Upcoming sections
 
-0.  [Programming basics](/tutorial/part-zero/): If you are new to programming, go here for an introduction to some basics like terminal, code editors, and browser consoles.
-1.  [Introduction to Gatsby basics](/tutorial/part-one/): Starting new projects, developing, and deploying sites.
-1.  [Introduction to using CSS in Gatsby](/tutorial/part-two/): Explore libraries like Typography.js and CSS Modules.
-1.  [Building nested layouts in Gatsby](/tutorial/part-three/): Layouts are sections of your site that are reused across multiple pages like headers and footers.
+0. [**Set up your development environment**](/tutorial/part-zero/): We’ll start by making sure you have installed all the tools you’ll need to begin development on a Gatsby site, and generate our very first site.
+1. [**Familiarize with Gatsby building blocks**](//tutorial/part-one/). We’ll tour the code we’ve generated for our new Gatsby site and get introduced to the Gatsby project code structure.
```

Comment:
My eyes kept re-reading this b/c I expected it to read "familiarize yourself"...which is a bit long. What about "Get to know Gatsby building blocks" or "Tour the default starter code"?